pkg/config/polling_manager.go

```diff
@@ -34,9 +34,6 @@ import (
 // DefaultPollingInterval sets default interval for polling manager
 const DefaultPollingInterval = 5 * time.Minute // default to 5 minutes for polling
 
-// DatafileURLTemplate is used to construct the endpoint for retrieving the datafile from the CDN
-const DatafileURLTemplate = "https://cdn.optimizely.com/datafiles/%s.json"
```

Comment:
This should stay here and be used to construct the URL of the request we make with the requester.